client/webserver/site/src/html/forms.tmpl

```diff
@@ -34,10 +34,10 @@
 
 {{define "newWalletForm"}}
 <div class="bg2 px-2 py-1 text-center fs18 position-relative">
-  Add a
+  [[[Add a]]]
   <img id="nwAssetLogo" class="micro-icon mx-1">
   <span id="nwAssetName"></span>
-  Wallet
+  [[[Wallet]]]
```

Comment:
Because of the the adjective-noun ordering issue between english and other languages, we might want to solve this one with a single `add_a_x_wallet` like was done with `create_a_x_wallet`

`"create_a_x_wallet": "Criar uma Carteira {{.Info.Name}}"`
(see [pt-BR PR](https://github.com/decred/dcrdex/pull/1185/files#diff-8a6dfc1ded985c870ad08435caba4a93af94beaa38c3b60c542331f6f0c7ac19R170))

vs english with

`"create_a_x_wallet": "Create a {{.Info.Name}} Wallet", // .Info.Name will be an asset symbol like DCR`

(although no template expressoins here, just html elements)

An alternative would be to change this entirely to use an article like with "Add a wallet **for** DCR" so the sentence structure doesn't (hopefully) change between languages.